viewing_party/party copy.py

```diff
@@ -0,0 +1,144 @@
+def create_movie(movie_title, genre, rating):
+    if movie_title and genre and rating:
+        new_movie = {"title": movie_title, "genre": genre, "rating": rating}
+        return new_movie
+    else:
+        return None
+
+def add_to_watched(user_data, movie):
+    user_data["watched"].append(movie)
+    return user_data
+
+def add_to_watchlist(user_data, movie):
+    user_data["watchlist"].append(movie)
+    return user_data
+
+def watch_movie(user_data, title):
+    for movie in user_data["watchlist"]:
+        if movie["title"] == title:
+            user_data["watched"].append(movie)
+            user_data["watchlist"].pop()
+    return user_data
+
+def get_watched_avg_rating(user_data):
+    ratings_list = []
+    for movie in user_data["watched"]:
+        ratings_list.append(movie["rating"])
+    if len(ratings_list) == 0:
+        return 0
+    else:
+        avg_rating = sum(ratings_list)/len(ratings_list)
+        return avg_rating
```

Comment:
Consider removing the `else`. In general if there is a way to write code without indentation, we prefer the version with less indentation